pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -277,6 +278,22 @@ message CommandPing {
 message CommandPong {
 }
 
+message CommandConsumerStats {
+        required uint64 request_id         = 1;
+        required string topic_name         = 2;
+        required string subscription_name  = 3;
+        required uint64 consumer_id        = 4;
+}
+
+message CommandConsumerStatsResponse {
+        required uint64 request_id              = 1;        
+        required string topic_name              = 2;
```

Comment:
Yes we can but a little more info might help in debug if we run into some trouble and since this call can be made at most once in 30 seconds - it won't harm the Network BW much.

Still if you feel strongly about it let me know - i will remove the fields